prime-angular-frontend/src/app/modules/site-registration/pages/care-setting-page/care-setting-page.component.ts

```diff
@@ -173,9 +173,6 @@ export class CareSettingPageComponent extends AbstractCommunitySiteRegistrationP
     if (this.formState.json.careSettingCode !== CareSettingEnum.DEVICE_PROVIDER) {
       this.siteFormStateService.deviceProviderFormState.clearIndividualDeviceProviders();
     }
-    if (this.formState.json.careSettingCode === CareSettingEnum.COMMUNITY_PHARMACIST) {
```

Comment:
Perhaps just a minor issue, but if SA selects PCHP, saves a Site ID, e.g. ABC, then comes back to Care Setting screen and switches to CP, for a short time, "ABC" can remain in the `Site` database row